src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = [...table.rows];
+
+rows.forEach((row) => {
+  const cellContent = row.cells[1].innerHTML;
+  const newCells = row.insertCell(row.cells.length - 1);
```

Comment:
The `insertCell` function inserts a cell at the specified position and returns a reference to the new cell. However, you're not using this new cell, but instead you're creating new cells in the next lines. This could lead to unexpected behavior. Consider creating and inserting the new cells in one step.